lib/metrics.py

```diff
@@ -348,3 +346,77 @@ def _get_db_conn(self):
                 psycopg2.extensions.ISOLATION_LEVEL_AUTOCOMMIT
             )
         return self.db
+
+
+class PaidAppsMetricsEmitterThread(BaseMetricsEmitterThread):
+    @property
+    def _select_stats_to_emit(self):
+        selected_stats = []
+        if buildpackutil.i_am_primary_instance():
+            selected_stats = [
+                self._inject_database_stats,
+                self._inject_storage_stats,
+                self._inject_health,
+            ]
+        selected_stats.append(self._inject_m2ee_stats)
+        return selected_stats
+
+    def _gather_metrics(self):
+        stats = {}
+        try:
+            for inject_method in self._select_stats_to_emit:
+                stats = inject_method(stats)
+        except psycopg2.OperationalError as up:
```

Comment:
```suggestion
        except psycopg2.OperationalError as exc:
```

I think the capture exception name `up` is somewhat confusing, especially further down in this block. Could you change it to the "usual" `exc` in accordance with the principle of least astonishment?